connectors/es/document.py

```diff
@@ -46,3 +49,27 @@ async def reload(self):
         self._seq_no = doc_source.get("_seq_no")
         self._primary_term = doc_source.get("_primary_term")
         self._source = doc_source.get("_source", {})
+
+    def debug(self, msg, *args, **kwargs):
+        logger.debug(self._prefix_msg(msg), *args, **kwargs)
+
+    def info(self, msg, *args, **kwargs):
+        logger.info(self._prefix_msg(msg), *args, **kwargs)
+
+    def warning(self, msg, *args, **kwargs):
+        logger.warning(self._prefix_msg(msg), *args, **kwargs)
+
+    def error(self, msg, *args, **kwargs):
+        logger.error(self._prefix_msg(msg), *args, **kwargs)
+
+    def exception(self, msg, *args, exc_info=True, **kwargs):
+        logger.exception(self._prefix_msg(msg), *args, exc_info=exc_info, **kwargs)
+
+    def critical(self, msg, *args, **kwargs):
+        logger.critical(self._prefix_msg(msg), *args, **kwargs)
+
+    def fatal(self, msg, *args, **kwargs):
+        logger.fatal(self._prefix_msg(msg), *args, **kwargs)
+
+    def _prefix_msg(self, msg):
```

Comment:
nit: Maybe a short doc string that overriding this method is what's expected (with an example), if you want to add prefixing/tracing capabilities would be helpful.